README.md

```diff
@@ -79,25 +69,45 @@
 
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
+**1.  Prerequisites**
 
 In order to run this project you need:
 
--VScode
--Prettier Extention
--Git
+- Visual Studio
+- Github
+- git
 
-### Setup
+**2. Setup**
 
 Clone this repository to your desired folder:
 
-Using the <em>git clone</em> command
+Example commands:
+
+git clone https://github.com/afsheen96/Hello-Microverse.git
+
+
+**3. Install**
+
+Install this project with: Vs code
+
+**4. Usage**
+ 
+ To run the project, use github Live Server extension
+ 
+
+**5. Run tests**
+ 
+run the test to make sure it is working properly.
+
+**6. Deployment**
+ 
+ now you can deploy your project.
+ 
```

Comment:
- [ ] Please try to follow up properly, as I mentioned in my previous review.  You must have to fill all the sections with relevant information. Kindly take a look at the below comparison table.

<table>
<tr>
<th align=“middle”>Your README</th>
<th align=“middle”>Professional README</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-30 23-11-14](https://github.com/Afsheen96/Hello-Microverse/assets/66476329/43d4bc90-2901-4b1d-97a0-79eb4433d266)


</td>
<td>

![Screenshot from 2023-05-08 18-55-11](https://github.com/Afsheen96/Hello-Microverse/assets/66476329/7d3c82b9-e9df-4c0b-9120-768e048ef172)

</td>
</tr>
</table>

Reply:
is anything else you want me to change except these two changing @skyv26

Reply:
@Afsheen96 Rest everything is perfect, just make this changes.